sklearn/cluster/dbscan_.py

```diff
@@ -103,7 +106,7 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     # neighborhood of point i. While True, its useless information)
     if metric == 'precomputed':
         D = pairwise_distances(X, metric=metric)
-        neighborhoods = [np.where(x <= eps)[0] for x in D]
+        neighborhoods = np.array([np.where(x <= eps)[0] for x in D])
```

Comment:
should be ``<`` as @ogrisel pointed out (that is where the inconsistency came from, right?)